src/components-examples/material/stepper/stepper-intl/stepper-intl-example.css

```diff
@@ -0,0 +1,17 @@
+.mat-stepper-horizontal {
```

Comment:
In our examples we like to target custom styles at demo classes so that its obvious it comes from the demo code, and not something that's baked into the component. So e.g. add `class="demo-stepper"` to `<mat-horizontal-stepper>` and target `.demo-stepper` here instead